src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  width: $card-width;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: $white;
+  border: 1px solid $grey-border;
+  border-radius: 5px;
+  padding: 16px;
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 16px auto 0;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $black;
+    margin-top: 40px;
+  }
+
+  &__product-code {
+    color: $grey-text;
+    margin-top: 4px;
+
+    &--font_size_10 {
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__reviews {
+    margin-top: 16px;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__reviews-text {
+    margin-top: 2px;
+    color: $black;
+
+    &--font_size_10 {
```

Comment:
Do you really need this modifier?
As I see all elements have this modifier so you can move modifier rules inside element class rules and remove modifier.